src/calc.py

```diff
@@ -18,3 +18,6 @@ def sub(a, b):
     ```
     """
     return a - b
+
+def mult(a,b):
```

Comment:
have we suggested `multiply` as a fn name?

```suggestion
def multiply(a,b):
```